hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -167,7 +167,7 @@
     private final MemberVersion version;
 
     @SuppressWarnings({"checkstyle:executablestatementcount", "checkstyle:methodlength"})
-    public Node(HazelcastInstanceImpl hazelcastInstance, Config staticConfig, NodeContext nodeContext) {
+    public Node(HazelcastInstanceImpl hazelcastInstance, Config staticConfig, NodeContext nodeContext) throws RuntimeException {
```

Comment:
Why declare the `RuntimeException`?

Reply:
I have no idea how that got there, good catch!